.gitignore

```diff
@@ -4,43 +4,41 @@
 .zip
 .gz
 .DS_Store
+.target
 .idea/
-dist/
-all-dependencies.txt
-self-modules.txt
-third-party-dependencies.txt
```

Comment:
I don't know why you remove these, these are aim to avoid mistakenly checking into codebase when running local check (they are generated when running local check)

Reply:
Because the dev branch has remove these lines, I just keep same with it and plan merge into dev and avoid conflicts

Reply:
Already restored